src/Microsoft.AspNet.Diagnostics.Entity/DatabaseErrorPageExtensions.cs

```diff
@@ -13,15 +14,35 @@ public static IApplicationBuilder UseDatabaseErrorPage([NotNull] this IApplicati
         {
             Check.NotNull(builder, nameof(builder));
 
-            return builder.UseDatabaseErrorPage(DatabaseErrorPageOptions.ShowAll);
+            return builder.UseDatabaseErrorPage(options => options.EnableAll());
         }
 
-        public static IApplicationBuilder UseDatabaseErrorPage([NotNull] this IApplicationBuilder builder, [NotNull] DatabaseErrorPageOptions options)
```

Comment:
This would be a good time to add doc comments for these. For some good examples of middleware doc comments, feel free to beg, borrow, and steal from a recent PR of mine: https://github.com/aspnet/Security/pull/529/files